code/modules/hydroponics/plant_genes.dm

```diff
@@ -315,11 +318,10 @@
 /datum/plant_gene/trait/noreact/on_new(obj/item/reagent_containers/food/snacks/grown/G, newloc)
 	..()
 	ENABLE_BITFIELD(G.reagents.flags, NO_REACT)
-
-/datum/plant_gene/trait/noreact/on_squash(obj/item/reagent_containers/food/snacks/grown/G, atom/target)
+	
+/datum/plant_gene/trait/noreact/on_prime(obj/item/reagent_containers/food/snacks/grown/G, atom/target)
```

Comment:
Why did you change the name of this?

Reply:
because I added a new on_prime proc. Squashing deletes the plant, and timers don't work unless they call a proc- the reason sepchems broke is because the plant was deleted before it mixed.